package.json

```diff
@@ -1531,6 +1531,15 @@
 							"altText": "create component from workspace folder"
 						}
 					},
+					{
+						"id": "createComponentFromGit",
+						"title": "Create Component from git repository",
+						"description": "Use the 'Create Component' UI and select 'From Existing Remote Git Repository' in order to import a project from git so that you can deploy it to OpenShift/Kubernetes.\n[Create Component](command:openshift.componentTypesView.registry.openHelmChartsInView)",
```

Comment:
The create component button calls `command:openshift.componentTypesView.registry.openHelmChartsInView`